include/openssl/ssl.h

```diff
@@ -2445,7 +2474,7 @@ void ERR_load_SSL_strings(void);
 # define SSL_R_WRONG_SSL_VERSION                          266
 # define SSL_R_WRONG_VERSION_NUMBER                       267
 # define SSL_R_X509_LIB                                   268
-# define SSL_R_X509_VERIFICATION_SETUP_PROBLEMS           269
+# define SSL_R_X509_VERIFICATION_SETUP_PROBLEMS           101
```

Comment:
What's this all about? It doesn't seem related to the other changes. Also this reason code has the original value in 1.0.2, so changing it doesn't seem like a good idea.